index.html

```diff
@@ -1,69 +1,85 @@
+<style>
+    body {
+        font-family: 'Rubik';
+    }
+</style>
 
-            <div class="vizual">
+<body>
 
-                <div class="logo">
-                    <img src="images/nike.svg" alt="logo Nike" width="65" height="24">
-                </div>
+    <div class="flex-container">
 
-                <p class="slogan">
-                    Zrychli tempo,<br>
-                    soustřeď se na rychlost!
-                </p>
+    </div>
 
-                <div class="bota">
-                    <img src="images/bota.png" alt="fotografie Nike Air Zoom Pegasus 33" width="415" height="295">
-                </div>
+    <div class="zbozi">
 
-                <div class="cenovka">
-                    <div class="cena">2450&nbsp;Kč</div>
-                    <div class="zaruka">Prodloužená záruka 3 roky</div>
-                </div>
+        <div class="vizual">
 
-            </div><!-- konec .vizual -->
+            <div class="logo">
+                <img src="images/nike.svg" alt="logo Nike" width="65" height="24">
+            </div>
 
+            <p class="slogan">
+                Zrychli tempo,<br>
+                soustřeď se na rychlost!
+            </p>
 
+            <div class="bota">
+                <img src="images/bota.png" alt="fotografie Nike Air Zoom Pegasus 33" width="300" height="208">
+            </div>
 
-            <div class="detaily">
+            <div class="cenovka">
+                <div class="cena">2450&nbsp;Kč</div>
+                <div class="zaruka">Prodloužená záruka 3 roky</div>
+            </div>
 
-                <h1 class="nazev">Nike Air Zoom Pegasus 33</h1>
+        </div><!-- konec .vizual -->
 
-                <p class="produkt">Dámská běžecká bota</p>
 
-                <p class="popis">
-                    Nike Air Zoom Pegasus 33 dokonale padne, dodá pocit rychlosti a má pružné tlumení, které ti umožní zrychlit tempo a soustředit se na rychlost.
-                </p>
 
-                <h2 class="parametr">Velikost</h2>
+        <div class="detaily">
 
-                <ul class="velikosti">
-                    <li>35</li>
-                    <li>36</li>
-                    <li>37</li>
-                    <li class="vybrana">38</li>
-                    <li>39</li>
-                    <li>40</li>
-                </ul>
+            <h1 class="nazev">NIKE AIR ZOOM PEGASUS 33</h1>
 
-                <div class="akce">
-                    <a class="tlacitko" href="#">Do košíku</a>
-                </div>
+            <p class="produkt">Dámská běžecká bota</p>
 
-            </div><!-- konec .detaily -->
+            <p class="popis">
+                Nike Air Zoom Pegasus 33 dokonale padne, dodá pocit rychlosti a má pružné tlumení, které ti umožní zrychlit tempo a soustředit se na rychlost.
+            </p>
 
+            <hr style="width:100%;text-align:left;margin-top:30px;margin-left:0;border-width: 1px;
+            border-style: solid;color:aquamarine">
```

Comment:
Ahoj, klidně bych víc stylovala ve stylech než v indexu, je to snazší a hodí se to mít pohromadě. Dnes už bývá html opravdu jen kostra. :)